plugins/cisco_umbrella_investigate/help.md

```diff
@@ -1440,6 +1501,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 3.1.5 - Added unittests
```

Comment:
Is it also ok to say we added output additional output examples in the help documentation?